content/en/docs/ibm/install-kubeflow.md

```diff
@@ -11,15 +11,15 @@ deploy Kubeflow on IBM Cloud.
 
 ### Installing the IBM Cloud developer tools
 
-> If you already have `ibmcloud` installed with the `ibmcloud cs` plugin, you
+> If you already have `ibmcloud` installed with the latest `ibmcloud ks` (Kubernetes Service) plug-in, you
 > can skip these steps.
 
 1.  Download and install the `ibmcloud` command line tool:
-    https://console.bluemix.net/docs/cli/index.html#overview
+    https://cloud.ibm.com/docs/cli?topic=cloud-cli-getting-started#overview
 
-1.  Install the `cs` (container-service) plugin:
+1.  Install the [Kubernetes Service plug-in](https://cloud.ibm.com/docs/cli?topic=containers-cli-plugin-kubernetes-service-cli):
```

Comment:
Thanks @ptitzler. 
Just a  few minor comments.
Should we use (or add) the url that point to actual command here?  
https://cloud.ibm.com/docs/cli?topic=cloud-cli-install-devtools-manually#idt-install-kubernetes-cli-plugin

Reply:
I was initially considering to do that but the link doesn't contain any useful information, aside from the command  that we are already documenting. The only other information in that section is a link to https://cloud.ibm.com/docs/containers?topic=containers-cli-plugin-kubernetes-service-cli, which is essentially the link I have already included.

That said, if you prefer this link I'll be happy to replace it.

Reply:
I see. Thanks!